src/status_im/data_store/realm/core.cljs

```diff
@@ -20,18 +20,18 @@
         (aset arr i (aget key i)))
       (.-buffer arr))))
 
-(defn realm-version
+(defn unencrypted-realm?
+  [file-name]
+  (boolean
+   (.schemaVersion rn-dependencies/realm file-name)))
```

Comment:
Can you detail this function? Not sure how version and encryption are related.

Reply:
We are just trying to open the file, to make sure it is unencrypted, basically we only care if it's successful or not.

Reply:
I can think of others reason for this call to fail than just being encrypted. Maybe we can find a better name? Or a safer method?